.github/workflows/ci.yaml

```diff
@@ -55,7 +55,10 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -e .[all]
+          pip install build
+          python -m build
+          WHEEL_FILE=$(ls dist/*.whl)
+          pip install $WHEEL_FILE[all]
```

Comment:
Was there a reason to install the wheel? In Cylc, some long time ago, we ended up with some messy code in one case where (if memory serves me well) the relative imports (`from .this.bla`) worked when installing with `pip install .` or `pip install $pkg`, but failed in the editable mode. Then we used the editable version in CICD to prevent that, and I never had the case where something worked in editable, but not with wheel/source via pip... so I just defaulted to always using editable in CICD... although maybe whatever issue we had is now long fixed in the python build tooling (which is newer).